packages/theme/src/react/index.js

```diff
@@ -3,6 +3,8 @@ import React from 'react'
 
 import * as vars from '../vars'
 
+export { default as withTheme } from './with-theme'
```

Comment:
Does this require extra babel plugin?  seems like exporting from an import is not supported in es6 modules properly atm.  Seems like there was/is a proposal as well, so I might be wrong.  I guess you're using it, so we've proved it's ok....  Mind just gut-checking my thought here?

Reply:
i think we're ok but i can look into the support

Reply:
if we wanted `default from` it's a new plugin but this seems to work 🤷‍♂️

Reply:
our current code works by compiling too

```
var _withTheme = require('./with-theme');

Object.defineProperty(exports, 'withTheme', {
  enumerable: true,
  get: function get() {
    return _interopRequireDefault(_withTheme).default;
  }
});
```